drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -160,16 +173,16 @@ def items_without_answers():
             "url_name": getattr(self, 'url_name', ''),
             "display_zone_labels": self.data.get('displayLabels', False),
             "display_zone_borders": self.data.get('displayBorders', False),
-            "items": items_without_answers(),
-            "title": self.display_name,
+            "items": items_without_answers(self),
+            "title": self._(self.display_name),
```

Comment:
Translating user-provided content doesn't just have the problem of missing translations; it can also cause problems. e.g. French instructor writes "Location" (which means "Rental" in French), but after getting translated by this code, it would appear in the course as "Emplacement" or something, which has a completely different meaning.